lusidtools/cocoon/seed_sample_data.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from lusidtools.cocoon.cocoon import load_from_data_frame
+from lusidtools.cocoon.utilities import load_json_file
+import logging
+
+logger = logging.getLogger()
+
+mappings = dict(load_json_file("config/seed_sample_data.json"))
+
+def seed_sample_data(api_factory, file_types, scope: str, transaction_source, mappings: dict = mappings):
+
+    """
+    Description:
+    This function allows users to seed their LUSID environment with some core data (e.g. instruments,
+    portfolios, and transaction) from one file.
+
+    :param lusid.utilities.ApiClientFactory api_factory: The api factory to use.
+    :param list[str] file_types: A list of the file_types for upload.
+    :param str scope: The scope of the transaction portfolio.
+    :param str transaction_source: a CSV file of data.
+    :param dict mappings: a file containing mapping of CSV or DataFrame headers to LUSID headers.
+    :return: various LUSID request responses.
+    """
+
+    data_frame = pd.read_csv(transaction_source)
+
+    def dynamic_load_from_data_frame(file_type):
+
+        return load_from_data_frame(
+            api_factory=api_factory,
+            file_type=file_type,
+            scope=scope,
+            data_frame=data_frame,
+            mapping_required=mappings[file_type]["required"],
+            mapping_optional=mappings[file_type]["optional"],
+            identifier_mapping=mappings[file_type]["identifier_mapping"],
+            property_columns=mappings[file_type]["properties"],
+            properties_scope=scope)
+
+    for file_type in file_types:
+
+        if file_type not in list(mappings.keys()):
+            raise ValueError(f"The provided file_type of {file_type} has no associated mapping")
+
+        logging.info(f"Loading {file_type} DataFrame into LUSID...")
+
+        dynamic_load_from_data_frame(file_type)
+
+        logging.info(f"Loading of {file_type} is COMPLETED")
```

Comment:
the function doesn't look like its returning anything (docstring says it is)

Reply:
You're right.
The function calls load_from_data_frame, but it doesn't return anything.
Code updated.